charts/fluent-bit/templates/_pod.tpl

```diff
@@ -103,14 +102,32 @@ containers:
     {{- if .Values.extraVolumeMounts }}
       {{- toYaml .Values.extraVolumeMounts | nindent 6 }}
     {{- end }}
-  {{- if .Values.extraContainers }}
-    {{- toYaml .Values.extraContainers | nindent 2 }}
-  {{- end }}
+{{- if .Values.hotReload.enabled }}
```

Comment:
Ah nice, so you have a watcher for config change that then triggers hot reload via localhost.

So we do not need the port but still need the webserver enabled, unless we can do it via signals? @cosmo0920

Reply:
Signals are a bad idea as they weaken the security model (and as I said above for Kubernetes the webserver should be running).